pacman-nodejs-app/docs/kubernetes-cluster-gke-federation.md

```diff
@@ -54,6 +54,17 @@ gcloud container clusters list
 export GCP_PROJECT=$(gcloud config list --format='value(core.project)')
 ```
 
+## Retrieve the Cluster credentials
+
+```
+gcloud config set container/use_client_certificate True
+export CLOUDSDK_CONTAINER_USE_CLIENT_CERTIFICATE=True
```

Comment:
@font no luck with using application_default_credentials instead of use_client_certificate. Without use_client_certificate, I get the following error when trying to initialize the federation:
```
kubefed init federation \
    --host-cluster-context=gke_${GCP_PROJECT}_us-west1-b_gce-us-west1 \
    --dns-provider='google-clouddns' --dns-zone-name=federation.com.
Error from server (Forbidden): User "jdetiber@redhat.com" cannot create roles.rbac.authorization.k8s.io in the namespace "federation-system".: "No policy matched.\nRequired \"container.roles.create\" permission." (post roles.rbac.authorization.k8s.io)

```